public/src/js/models/collections/collection.js

```diff
@@ -93,6 +93,25 @@ export default class Collection extends BaseClass {
         this.setPending(true);
         this.loaded = this.load().then(() => onDomLoad);
         this.state.visibleCount({});
+
+
+        this.lastAlertSent = ko.pureComputed (() => {
```

Comment:
can we keep the constructor as small as possible?

`this.lastAlertSent = ko.pureComputed(this.getLastAlertTime, this)`

and then the logic is implemented in that function